packages/dataviews/src/components/dataviews-picker/stories/index.story.tsx

```diff
@@ -0,0 +1,251 @@
+/**
+ * External dependencies
+ */
+import type { Meta } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState, useMemo, useCallback, useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import DataViewsPicker from '../index';
+import {
+	data,
+	fields,
+	type SpaceObject,
+} from '../../dataviews/stories/fixtures';
+import { LAYOUT_PICKER_GRID } from '../../../constants';
+import { filterSortAndPaginate } from '../../../filter-and-sort-data-view';
+import type { Action, View } from '../../../types';
+
+const meta = {
+	title: 'DataViews/DataViewsPicker',
+	component: DataViewsPicker,
+} as Meta< typeof DataViewsPicker >;
+
+export default meta;
+
+export const Default = ( {
+	perPageSizes = [ 10, 25, 50, 100 ],
+	isMultiselectable,
+	isGrouped,
+	infiniteScrollEnabled,
+}: {
+	perPageSizes: number[];
+	isMultiselectable: boolean;
+	isGrouped: boolean;
+	infiniteScrollEnabled: boolean;
+} ) => {
+	const [ view, setView ] = useState< View >( {
+		type: LAYOUT_PICKER_GRID,
+		label: 'Galactic Bodies',
```

Comment:
I'm thinking that `label` makes more sense as a DataViewsPicker prop. My rationale is that the view object reflects the current view state, things that can change upon user interactions. My understanding is that `label` never changes, it's a property of the data that we use to add an accessible label to the item list.